Topl/Bifrost
# Chain replicator

## Purpose
The chain replicator takes info about new blocks from NodeViewHolder and updates a MongoDB database for block explorer, it also figures out the missing blocks on MongoDB and send them periodically over to the database

## Approach
- Added a chain replicator typed actor

## Tests
- Manually tested updating the database:
	- started node without chain replicator turned on
	- run node with existing blocks and have chain replicator turned on
	- run node with existing blocks and have chain replicator and check missing blocks turned on

## Closes
- #1476
